build.sbt

```diff
@@ -36,13 +37,11 @@ inThisBuild(
 
 ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
 
-lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+lazy val core = crossProject(JSPlatform, JVMPlatform /*, NativePlatform*/ )
```

Comment:
Is there a reason for commenting native version?

Reply:
Yes, because I had to bump to Scala 3.2.0-RC2 the Native plugin is not yet published for these versions.

Reply:
CI for for native can be disabled (for a while) to not block this PR

Reply:
@kpodsiad cool, thanks, I've fixed formatting and disabled Native CI for now.